common/taskana-common-security/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>taskana-common-security</artifactId>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>The global helper module</description>
```

Comment:
I'd prefer something like "The helper module for security utils"...copy paste issue here?

Reply:
yep.